internal/command/scale/count.go

```diff
@@ -0,0 +1,93 @@
+package scale
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/superfly/flyctl/client"
+	"github.com/superfly/flyctl/internal/appconfig"
+	"github.com/superfly/flyctl/internal/command"
+	"github.com/superfly/flyctl/internal/flag"
+	"github.com/superfly/flyctl/iostreams"
+)
+
+func newScaleCount() *cobra.Command {
+	const (
+		short = "Change an app's VM count to the given value"
+		long  = `Change an app's VM count to the given value.
+
+For pricing, see https://fly.io/docs/about/pricing/`
+	)
+	cmd := command.New("count", short, long, runScaleCount,
+		command.RequireSession,
+		command.RequireAppName,
+		failOnMachinesApp,
+	)
+	cmd.Args = cobra.MinimumNArgs(1)
+	flag.Add(cmd,
+		flag.App(),
+		flag.AppConfig(),
+		flag.Int{Name: "max-per-region", Description: "Max number of VMs per region", Default: -1},
+	)
+	cmd.AddCommand()
```

Comment:
[AFAIK](https://github.com/spf13/cobra/blob/main/command.go#L1255) calling this with no arguments doesn't do anything. Safe to remove this (and the same thing in `memory`, `show`, and `vm`)?

Reply:
good catch. fixing

Reply:
296147e9 fixed it